src/utils/tutorial/index.js

```diff
@@ -0,0 +1,30 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import StringConstant from '../string/StringConstant';
+import {TUTOR_CREATE_POST_ANON, TUTOR_POST_ANON} from '../constants';
+
+// eslint-disable-next-line consistent-return
+export const getStorageKey = async (name) => {
+  switch (name) {
+    case StringConstant.tutorialAnonymousPostTitle:
+      return TUTOR_POST_ANON;
+    case StringConstant.tutorialCreateAnonymousPostTitle:
+      return TUTOR_CREATE_POST_ANON;
+    default:
+  }
+};
+
+export const setFinishedTutor = async (key) => {
+  // await AsyncStorage.setItem(key, true);
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `setFinishedTutor` is commented out. If this function is not used, it should be removed to avoid confusion. If it's intended to be used in the future, consider adding a comment explaining why it's currently commented out.

```diff
-  // await AsyncStorage.setItem(key, true);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->